dapp/src/utils/cardano/datum.ts

```diff
@@ -0,0 +1,34 @@
+import { Constr, Data } from "lucid-cardano";
+
+export const getLockDatum = ({
+  ownerPaymentKeyHash,
+}: {
+  ownerPaymentKeyHash: string;
+}): string => {
+  const data = new Constr(0, [

```

Comment:
Probably we should have a comment for both these files explaining the magic numbers (0 and 1) here